Word-document/Merge-documents-with-same-margins/.NET-Standard/Merge-documents-with-same-margins/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Merge_documents_with_same_margins
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Open the file as a stream.
+            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            {
+                //Load the file stream into a Word document.
+                using (WordDocument sourceDocument = new WordDocument(sourceStreamPath, FormatType.Automatic))
+                {
+                    using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    {
+                        //Open the destination document.
+                        using (WordDocument destinationDocument = new WordDocument(destinationStreamPath, FormatType.Automatic))
+                        {
+                            //Get the page setup of the destination document.
+                            WPageSetup destinationDocumentPageSetup = destinationDocument.LastSection.PageSetup;
+                            //Iterate source document.
+                            foreach (WSection sourceSection in sourceDocument.Sections)
+                            {
+                                sourceSection.PageSetup.Margins = destinationDocumentPageSetup.Margins;
+                                //Clone the source document sections to the destination document.
```

Comment:
@sharonaumao - Please modify the XML comments properly.

Reply:
@Ramaraj-Marimuthu -I have modified